features.md

```diff
@@ -0,0 +1,335 @@
+# <a name="features" />Features Document
+
+A [runtime](glossary.md#runtime) MAY provide a JSON document about its implemented features to [runtime callers](glossary.md#runtime-caller).
+This JSON document is called ["features document"](glossary.md#features-document).
+
+The features document is irrelevant to the actual availability of the features in the host operating system.
```

Comment:
> Is it the role of the caller of the oci container runtime to ensure that it is actually available on the actual host? 

Yes

>  I felt it would be nice to check if the feature is even available on the host. What do you think?

Out of the scope of this PR (and I don't see necessity)
https://github.com/opencontainers/runtime-spec/pull/1130#discussion_r770459768